codejail_service/apps/api/v0/views.py

```diff
@@ -120,19 +120,19 @@ def code_exec(request):
     if unsafely:
         return Response({'error': "Refusing codejail execution with unsafely=true"}, status=400)
 
-    try:
-        # This wrapped version of safe_exec doesn't mutate the globals dict
-        output_globals_dict = safe_exec(
-            complete_code,
-            input_globals_dict,
-            python_path=python_path,
-            extra_files=extra_files,
-            limit_overrides_context=limit_overrides_context,
-            slug=slug,
-        )
-    except BaseException as e:
-        log.debug("Codejail execution failed for {slug=} with: {e}")
-        return Response({'emsg': str(e)})
-
-    log.debug("Codejail execution succeeded for {slug=}, with globals={output_globals_dict!r}")
-    return Response({'globals_dict': output_globals_dict})
+    # This wrapped version of safe_exec doesn't mutate the globals dict
+    (globals_out, emsg) = safe_exec(
```

Comment:
[idea] Wonder about using `error_message` until we get to the response, to not break the api, but to make the code easier to read. Thoughts?

Reply:
Sure, will change to `error_message`.